recipes/libsigcpp/2.x.x/conanfile.py

```diff
@@ -1,20 +1,23 @@
-from conans import ConanFile, Meson, tools
-from conan.tools.files import rename
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.tools.apple import fix_apple_shared_install_name
+from conan.tools.build import check_min_cppstd
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.files import copy, get, rename, replace_in_file, rm, rmdir
+from conan.tools.layout import basic_layout
+from conan.tools.meson import Meson, MesonToolchain
 import glob
 import os
-import shutil
 
-required_conan_version = ">=1.43.0"
+required_conan_version = ">=1.53.0"
 
 
-class LibSigCppConanV2(ConanFile):
+class LibSigCppConan(ConanFile):
     name = "libsigcpp"
     homepage = "https://github.com/libsigcplusplus/libsigcplusplus"
     url = "https://github.com/conan-io/conan-center-index"
     license = "LGPL-3.0"
```

Comment:
Could you update the license to be either `LGPL-3.0-only` or `LGPL-3.0-or-later` here? The `LGPL-3.0` SPDX identifier is deprecated.

Reply:
LGPL-3.0-only seems to be the appropriate equivalent